yt_dlp/YoutubeDL.py

```diff
@@ -618,6 +619,12 @@ def __init__(self, params=None, auto_init=True):
                 '         If you experience any issues while using this option, '
                 f'{self._format_err("DO NOT", self.Styles.ERROR)} open a bug report')
 
+        if self.params.get('enable_file_urls'):
+            self.report_warning(
+                f'You have enabled support for file:// URLs. '
+                'These are disabled by default in yt-dlp for security reasons. '
+                f'{self._format_err("Use at your own risk.", self.Styles.ERROR)}')
+
```

Comment:
This isnt needed imo. The security risk is because servers may allow users to pass arbitrary URLs to yt-dlp. If the server allows arbitrary options, there are other options (like `--exec`) which are far more concerning